lib/circuit.js

```diff
@@ -381,6 +412,19 @@ class CircuitBreaker extends EventEmitter {
     return this[STATUS].stats;
   }
 
+  // This is only for convience
+  exportState () {
+    return {
+      enabled: this.enabled,
+      closed: this.closed,
+      open: this.opened,
+      halfOpen: this.halfOpen,
+      warmUp: this.warmUp,
+      pendingClose: this.pendingClose,
+      shutdown: this.isShutdown
+    };
+  }
```

Comment:
Barring my comments above about field/behavior duplication, I like this. It got me thinking that a [`toJSON()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#tojson_behavior) method might be useful. E.g.

```suggestion
  exportState () {
    return this.toJSON();
  }
  
  // ensures that when a circuit is serialized as JSON, it looks like this
  toJSON() {
    return {
      name: this.name,
      group: this.group,
      enabled: this.enabled,
      opened: this.opened,
      halfOpen: this.halfOpen,
      warmUp: this.warmUp,
      shutdown: this.isShutdown,
      status: this.status // obviously this would affect the ctor params
    };
  }
```
This also ensures that a circuit's state and status on import/export are in sync.